website/common/locales/en/groups.json

```diff
@@ -258,5 +258,9 @@
   "paymentDetails": "Payment Details",
   "aboutToJoinCancelledGroupPlan": "You are about to join a group with a canceled plan. You will NOT receive a free subscription.",
   "cannotChangeLeaderWithActiveGroupPlan": "You can not change the leader while the group has an active plan.",
-  "leaderCannotLeaveGroupWithActiveGroup": "A leader can not leave a group while the group has an active plan"
+  "leaderCannotLeaveGroupWithActiveGroup": "A leader can not leave a group while the group has an active plan",
+  "youHaveGroupPlan": "You have a free subscription because you are a member of a group that has a Group Plan. This will end when you  are no longer in the group that has a group plan. Any extra months you have will be applied at the end of the group plan.",
+  "cancelGroupSub": "Cancel Group Plan",
+  "confirmCancelGroupPlan": "Are you sure you want to cancel the group plan and remove its benefits from all members?",
```

Comment:
"Are you sure you want to cancel the group plan and remove its benefits from all members, including their free subscriptions?"

I'd like to make that totally clear to them.